backend/danswer/configs/chat_configs.py

```diff
@@ -75,6 +75,8 @@
     or "The name of the conversation must be in the same language as the user query."
 )
 
```

Comment:
Let's add a comment for what this does and why someone would want to turn it on/off. Also let's include this across the board for all of the deployment options (you can refer to DISABLE_LLM_CHUNK_FILTER to see where env vars are set across deployments)

Reply:
Comment could be like:

Agentic search takes significantly more tokens and therefore has much higher cost. This configuration allows users to get a search-only experience with instant results and no involvement from the LLM.

Reply:
Sounds great! Renamed here and in the docs